modules/configuring-sinkbinding-with-ossm.adoc

```diff
@@ -0,0 +1,159 @@
+:_content-type: PROCEDURE
+[id="configuring-sinkbinding-with-ossm_{context}"]
+= Configuring SinkBinding with {SMProductShortName}
+
+This procedure describes how to configure SinkBinding with {SMProductShortName}.
+
+.Prerequisites
+
+* You have set up integration of {SMProductShortName} and {ServerlessProductShortName}.
+
+.Procedure
+
+. Create a `Service` in a namespace that is member of the `ServiceMeshMemberRoll`:
++
+.Example `event-display-service.yaml` configuration file
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+  namespace: <namespace> <1>
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true" <2>
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+    spec:
+      containers:
+      - image: quay.io/openshift-knative/knative-eventing-sources-event-display:latest
+----
+<1> A namespace that is a member of the `ServiceMeshMemberRoll`.
+<2> This annotation injects {SMProductShortName} sidecars into the Knative service pods.
+
+. Apply the `Service` resource:
```

Comment:
```suggestion
. Apply the `Service` object:
```
Just for consistency with other parts of the PR, though switching everything from "object" to "resource" is also fine.